backend/dataall/modules/s3_datasets_shares/services/s3_share_service.py

```diff
@@ -169,9 +178,14 @@ def reapply_share_items_for_dataset(uri: str):
     def list_shared_tables_by_env_dataset(dataset_uri: str, env_uri: str):
         context = get_context()
         with context.db_engine.scoped_session() as session:
+            log.info(
+                S3ShareObjectRepository.query_dataset_tables_shared_with_env(
+                    session, env_uri, dataset_uri, context.username, context.groups
+                )
+            )
             return [
-                {'tableUri': t.tableUri, 'GlueTableName': t.GlueTableName}
-                for t in S3ShareObjectRepository.query_dataset_tables_shared_with_env(
+                {'tableUri': res[0], 'GlueTableName': res[1] + (f'_{res[2]}' if res[2] else '')}
```

Comment:
why do we need to access fields by index, cannot we directly unpack the fields with dot notation?

Reply:
wanted to do this before but was having difficulty so went by index and meant to re-visit

have updated now by key name

Reply:
checked! all good!